.storybook/stories/alert/alert.stories.ts

```diff
@@ -33,14 +33,30 @@ const defaultStory: Story = args => ({
   props: { ...args },
 });
 
+const labeledAlertTypes = ALERT_TYPES.reduce((a, val) => {
+  let innerVal = val;
+  if (innerVal === 'loading' || innerVal === 'unknown') {
+    innerVal += ' - only works with "clrAlertLightweight" set to TRUE';
+  }
+  return { ...a, [val]: innerVal };
```

Comment:
```suggestion
  a[val] = innerVal;
  return a;
```

This is not really an issue here since the array length is small and it only happens once, but in general, a new object should not be created in every iteration when using `reduce`.

Reply:
Corrected in commit bc7eabe